src/App.js

```diff
@@ -22,15 +57,23 @@ function App() {
         >
           Learn React
         </a>
-        <Button buttonType={"primary"} onClickFunction={alertBrowser}>This is primary button</Button>
-        <Button buttonType={"secondary"} onClickFunction={alertBrowser}>This is secondary button</Button>
-        <Button isDisabled onClickFunction={alertBrowser}>This is disabled button</Button>
-      </header>
-      <table>
-        <td>
-          <tr>Hi</tr>
-        </td>
-      </table>
+        </header><br/><br/>
+        <body>
+          {inputs.map((input) => (
```

Comment:
This is good, but I think we might avoid using `map()` to render these inputs since their styling (specifically width) and validation functions will be different. No actions needed here tho!